FBTweak/_FBTweakCollectionViewController.m

```diff
@@ -28,11 +30,22 @@ - (instancetype)initWithTweakCategory:(id<FBTweakCategory>)category
     _tweakCategory = category;
     self.clearsSelectionOnViewWillAppear = YES;
     self.title = _tweakCategory.name;
-    self.refreshControl = [[UIRefreshControl alloc] init];
-    [self.refreshControl addTarget:self
-                            action:@selector(_reloadData)
-                  forControlEvents:UIControlEventValueChanged];
-    [self _reloadData];
+
+    _kvoController = [FBKVOController controllerWithObserver:self];
+    [_kvoController observe:_tweakCategory keyPath:@"tweakCollections"
+                        options:NSKeyValueObservingOptionInitial | NSKeyValueObservingOptionNew
+                          block:^(id observer, id object, NSDictionary *change) {
```

Comment:
Why the compiler doesn't complain that `observer`, `object` and `change` aren't used?

Reply:
I'm guessing this isn't enabled in this project.